### ISSUE TITLE
lastestUrl doesn't work anymore

### ISSUE THREAD

**Issue by KristjanESPERANTO** [zhouxichun/MMM-TiHimawari8Earth#4]
The URL https://himawari8-dl.nict.go.jp/himawari8/img/FULL_24h/latest.json doesn't work anymore. So the module doesn't display anything :slightly_frowning_face:

**Comment by sephiran** [comment]
yes.. on the himawari website it says "The data download service for Himawari satellite data via the web application will be discontinued as of July 1, 2024"..

**Comment by zhouxichun** [comment]
It seems to be accessible again
